docs/introduction/index.md

```diff
@@ -1,26 +1,255 @@
 ---
-title: Why we created Spine
+title: Development Process
 headline: Documentation
 bodyclass: docs
 layout: docs
-type: markdown
 ---
+# Development Process
 
-# Why we created Spine
+<p class="lead">Building a solution based on Spine Event Engine framework is an iterative process
+which consists of the stages described in this document.</p>
+ 
+## Getting domain knowledge
+
+The purpose of this step is to find out what we're going to build and why.
+Consider using [EventStorming](https://eventstorming.com) or other DDD-based analysis 
+methodology for helping you grasp the domain knowledge from the experts.
+ 
+Most likely that the solution would have several [Bounded Contexts](concepts.html#bounded-context). 
+For each context developers need to define:
+  * <strong>Signals</strong>
+    - [Events](concepts.html#event)
+    - [Commands](concepts.html#command)
+    - [Rejections](concepts.html#rejection)
+  * <strong>Entities</strong>  
+    - [Aggregates](concepts.html#aggregate) 
+    - [Process Managers](concepts.html#process-manager)
+    - [Projections](concepts.html#projection).
+
+It is likely that some of the bits of this picture would change during the process.
+But the whole team, including domain experts, need to have complete understanding of how the 
+business works to avoid “surprises” down the road. 
+
+We return to learning the domain when we discover inconsistencies in the model,
+or we need more information about how the business works, or the business wants to develop further
+and we need to update the model.
+
+Once we got enough domain knowledge we proceed to the implementation. 
```

Comment:
Do we want to mention #EventStorming before this sentence?

Reply:
We already mention in when we start the domain knowledge section. There's the link in the first paragraph of the section.